sdk/core/Azure.Core.Experimental/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+- Removed the `PopTokenRequestContext` type and added the proof of posession related properties to `TokenRequestContext` in Azure.Core.
```

Comment:
> 
```suggestion
- Removed the `PopTokenRequestContext` type and added the proof of possession related properties to `TokenRequestContext` in Azure.Core.
```

Reply:
Question on this: @jsquire asked us not to list breaking changes in this section for preview  libraries.  Since Experimental is perpetually beta, should this go under the "other" section as we do in other libraries?